src/sentry/static/sentry/app/views/settings/components/scrollToTop.jsx

```diff
@@ -5,10 +5,16 @@ import React from 'react';
 class ScrollToTop extends React.Component {
   static propTypes = {
     location: PropTypes.object,
+    disable: PropTypes.func,
   };
 
   componentDidUpdate(prevProps) {
-    if (this.props.location !== prevProps.location) {
+    const {disable, location} = this.props;
+
+    const shouldDisable =
+      typeof disable === 'function' && disable(location, prevProps.location);
```

Comment:
You might be able to use `callIfFunction()` here. ⛳

Reply:
oh TIL, thanks!